contracts/src/utils/context.rs

```diff
@@ -0,0 +1,17 @@
+//! Context functions used for mocking unit tests.
+
+use stylus_sdk::{alloy_primitives::Address, msg};
+
+/// Returns the address of the message sender.
+#[cfg(test)]
+pub fn msg_sender() -> Address {
+    motsu::prelude::Context::current()
+        .get_msg_sender()
+        .expect("msg_sender should be set")
+}
+
+/// Returns the address of the message sender.
+#[cfg(not(test))]
+pub fn msg_sender() -> Address {
+    msg::sender()
+}
```

Comment:
Here is an example how `msg_sender` can be called just for test cases. This design is not perfect since motsu users should implement this mock for their own library manually or through macro.
Calling stylus sdk api `msg::sender()` won't work due to [this limitation](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/30#issuecomment-2037231720).